### PR TITLE
1398 spanish localize nodes

### LINKED ISSUE #1398: [Spanish] localize 'Nodes'
Localize the `content/en/nodes.md` document to Spanish

### PR DESCRIPTION
**Pull request by mdf-ido**
### Describe your changes

Adding the definition of Nodes in Spanish. 

### Related issue number or link (ex: `resolves #issue-number`)

Resolves #1398 

### Checklist before opening this PR (put `x` in the checkboxes)
- [ x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [ x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/nodes.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Nodos
+status: Completed
+category: Tecnología
+tags: ["arquitectura"]
```
Comment (electrocucaracha) [inline content/es/nodes.md:5]:
```suggestion
tags: ["architecture"]
```

File: content/es/nodes.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+## ¿Qué es?
+
+Un nodo es un servidor que trabaja en conjunto a otros servidores, o nodos, para realizar una tarea común. Por ejemplo, tu laptop, el módem y la impresora están conectados a través de tu red wifi comunicándose y colaborando, y cada uno representa un nodo. En [computación en la nube](/cloud-computing/), un nodo puede ser un servidor físico, un servidor virtual, denominado [VM](/virtual-machine/), o incluso un [contenedor](/container/).
```
Comment (electrocucaracha) [inline content/es/nodes.md:10]:
```suggestion
Un nodo es un servidor que trabaja en conjunto a otros servidores, o nodos, para realizar una tarea común. Por ejemplo, tu laptop, el módem y la impresora están conectados a través de tu red wifi comunicándose y colaborando, y cada uno representa un nodo. En [computación en la nube](/es/cloud-computing/), un nodo puede ser un servidor físico, un servidor virtual, denominado [VM](/virtual-machine/), o incluso un [contenedor](/container/).
```

File: content/es/nodes.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+## Problema que aborda
+
+Una aplicación podría (y muchas lo hacen) ejecutarse en una sola máquina, esto conlleva algunos riesgos como el fallo del sistema subyacente interrumpira la aplicación. Para solucionar esto, los desarrolladores comenzaron a crear [aplicaciones distribuidas](/distributed-apps/) donde cada proceso se ejecuta en su propio nodo. Así, los nodos ejecutan aplicaciones o procesos como parte de un grupo que forma un [cluster](/cluster/), o grupo, de nodos que trabajan juntos para lograr un objetivo común.
```
Comment (electrocucaracha) [inline content/es/nodes.md:14]:
```suggestion
Una aplicación podría (y muchas lo hacen) ejecutarse en una sola máquina, esto conlleva algunos riesgos como el fallo del sistema subyacente interrumpira la aplicación. Para solucionar esto, los desarrolladores comenzaron a crear [aplicaciones distribuidas](/distributed-apps/) donde cada proceso se ejecuta en su propio nodo. Así, los nodos ejecutan aplicaciones o procesos como parte de un grupo que forma un [clúster](/es/cluster/), o grupo, de nodos que trabajan juntos para lograr un objetivo común.
```

File: content/es/nodes.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+## ¿Cómo ayuda?
+
+Un nodo es una unidad de cálculo distinta (memoria, CPU, red) que se puede asignar a un clúster. En una plataforma o aplicación [nativa de la nube](/cloud-native-tech/) un nodo representa una única unidad que puede realizar trabajo. Idealmente, los nodos individuales son indiferenciados en el sentido de que cualquier nodo de un tipo particular es indistinguible de cualquier otro nodo del mismo tipo.
```
Comment (electrocucaracha) [inline content/es/nodes.md:18]:
```suggestion
Un nodo es una unidad de cálculo distinta (memoria, CPU, red) que se puede asignar a un clúster. En una plataforma o aplicación [nativa para la nube](/es/cloud-native-tech/) un nodo representa una única unidad que puede realizar trabajo. Idealmente, los nodos individuales son indiferenciados en el sentido de que cualquier nodo de un tipo particular es indistinguible de cualquier otro nodo del mismo tipo.
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]

File: content/es/nodes.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+---
+title: Nodos
+status: Completed
+category: Tecnología
+tags: ["architecture"]
+---
+
+## ¿Qué es?
+
+Un nodo es un servidor que trabaja en conjunto a otros servidores, o nodos, para realizar una tarea común. Por ejemplo, tu laptop, el módem y la impresora están conectados a través de tu red wifi comunicándose y colaborando, y cada uno representa un nodo. En [computación en la nube](/es/cloud-computing/), un nodo puede ser un servidor físico, un servidor virtual, denominado [VM](/virtual-machine/), o incluso un [contenedor](/container/).
```
Comment (krol3) [inline content/es/nodes.md:10]:
```suggestion
Un nodo es un servidor que trabaja en conjunto a otros servidores, o nodos, para realizar una tarea común. Por ejemplo, tu laptop, el módem y la impresora están conectados a través de tu red wifi comunicándose y colaborando, cada uno representa un nodo. En [computación en la nube](/es/cloud-computing/), un nodo puede ser un servidor físico, un servidor virtual, denominado [VM](/virtual-machine/), o incluso un [contenedor](/container/).
```

File: content/es/nodes.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+## Problema que aborda
+
+Una aplicación podría (y muchas lo hacen) ejecutarse en una sola máquina, esto conlleva algunos riesgos como el fallo del sistema subyacente interrumpira la aplicación. Para solucionar esto, los desarrolladores comenzaron a crear [aplicaciones distribuidas](/distributed-apps/) donde cada proceso se ejecuta en su propio nodo. Así, los nodos ejecutan aplicaciones o procesos como parte de un grupo que forma un [clúster](/es/cluster/), o grupo, de nodos que trabajan juntos para lograr un objetivo común.
```
Comment (krol3) [inline content/es/nodes.md:14]:
```suggestion
Una aplicación podría (y muchas lo hacen) ejecutarse en una sola máquina, esto conlleva algunos riesgos como el fallo del sistema subyacente interrumpirá la aplicación. Para solucionar esto, los desarrolladores comenzaron a crear [aplicaciones distribuidas](/distributed-apps/) donde cada proceso se ejecuta en su propio nodo. Así, los nodos ejecutan aplicaciones o procesos como parte de un grupo que forma un [clúster](/es/cluster/), o grupo, de nodos que trabajan juntos para lograr un objetivo común.
```

**Review by krol3 — CHANGES_REQUESTED** [review]
Nice work!

**Comment by electrocucaracha** [comment]
@mdf-ido podrías agregar `VM` en la lista de palabras para poder aprobar este cambio?

**Review by electrocucaracha — APPROVED** [review]

**Comment by electrocucaracha** [comment]
@krol3 podrías revisar nuevamente estos cambios?

**Comment by mdf-ido** [comment]
@electrocucaracha - Todo bien con esto o necesita alguna actualización de mi parte?

**Comment by electrocucaracha** [comment]
@mdf-ido listo, ya lo agregue al branch *dev-es* y en algún momento pondremos todos los cambios en *main* muchas gracias

**Pull request merged**